### PR TITLE
Update AutoMM examples for tabular data

### PR DESCRIPTION
**Pull request by yiqings**
*Issue #, if available:*

*Description of changes:*

1. Refine the datasets used in the examples (remove the redundant index column). 
2. Update the results provided in the tabular examples.
3. Add `multimodal/src/autogluon/multimodal/version.py` to `.gitignore`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/automm/tabular_dl/README.md [hunk 1]
```diff
@@ -61,13 +61,35 @@ problem_type | regression | binary | multiclass | multiclass | binary | multicla
 #classes | - | 2 | 100 | 4 | 2 | 1000 | 2 | - | 7 | - | -
 Best in [1] | 0.459 | 0.859 | 0.396 | 0.732 | 0.729 | 0.963 | 0.8982 | 8.794 | 0.970 | 0.753 | 0.745
 FT-Transformer in [1] | 0.459 | 0.859 | 0.391 | 0.732 | 0.729 | 0.960 | 0.8982 | 8.855 | 0.970 | 0.756 | 0.746
-AutoMM FT-Transformer + Const Lr=1e-4 | 0.404 | 0.863 | 0.388 | 0.727 | 0.729 | 0.954 | OverflowError | 0.781 | 0.964 | AttributeError | 0.904
-AutoMM FT-Transformer + Cosine Lr=1e-4| 0.401 | 0.864 | 0.388 | 0.728 | 0.729 | 0.952 | OverflowError | 0.780 | 0.964 | 0.765 | 0.924
+AutoMM FT-Transformer | 0.482 | 0.859 |  |  |  | 0.949 | OverflowError |  | 0.963 |  | 0.761
 
-The trained weights and configs are accessible at https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/results.zip or s3://autogluon/results/tabular/results.zip. Use
+The `AutoMM FT-Transformer` column is runed by `run_all.sh`, where we use the following hyperparameter:
```
Comment (sxjscience) [inline examples/automm/tabular_dl/README.md:66]: You can reproduce the `AutoMM FT-Transformer` column by running `run_all.sh`, where we use the following hyperparameter:

File: multimodal/src/autogluon/multimodal/version.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+"""This is the autogluon version file."""
+__version__ = '0.5.0b20220623'
```
Comment (sxjscience) [inline multimodal/src/autogluon/multimodal/version.py:2]: Remove this file.

File: examples/automm/tabular_dl/README.md (hunk 1 above)
Comment (yiqings) [reply examples/automm/tabular_dl/README.md:66]: Great, thanks for the suggestion.

File: examples/automm/tabular_dl/README.md [hunk 3]
```diff
@@ -61,13 +61,35 @@ problem_type | regression | binary | multiclass | multiclass | binary | multicla
 #classes | - | 2 | 100 | 4 | 2 | 1000 | 2 | - | 7 | - | -
 Best in [1] | 0.459 | 0.859 | 0.396 | 0.732 | 0.729 | 0.963 | 0.8982 | 8.794 | 0.970 | 0.753 | 0.745
 FT-Transformer in [1] | 0.459 | 0.859 | 0.391 | 0.732 | 0.729 | 0.960 | 0.8982 | 8.855 | 0.970 | 0.756 | 0.746
-AutoMM FT-Transformer + Const Lr=1e-4 | 0.404 | 0.863 | 0.388 | 0.727 | 0.729 | 0.954 | OverflowError | 0.781 | 0.964 | AttributeError | 0.904
-AutoMM FT-Transformer + Cosine Lr=1e-4| 0.401 | 0.864 | 0.388 | 0.728 | 0.729 | 0.952 | OverflowError | 0.780 | 0.964 | 0.765 | 0.924
+AutoMM FT-Transformer | 0.482 | 0.859 | 0.379 | 0.721 | 0.726 | 0.949 | OverflowError |  | 0.963 |  | 0.761
 
-The trained weights and configs are accessible at https://autogluon.s3.us-west-2.amazonaws.com/results/tabular/results.zip or s3://autogluon/results/tabular/results.zip. Use
+You can reproduce the `AutoMM FT-Transformer` column by running `run_all.sh`, where we use the following hyperparameter:
```
Comment (sxjscience) [inline examples/automm/tabular_dl/README.md:66]: Actually, just noticed that it should be "row" rather than "column".

**Comment by szha** [comment]
Job PR-1893-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1893/2/index.html

**Comment by szha** [comment]
Job PR-1893-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1893/3/index.html

**Review by sxjscience — APPROVED** [review]
lgtm

**Comment by yiqings** [comment]
Hi @sxjscience, this PR is ready.

**Comment by szha** [comment]
Job PR-1893-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1893/5/index.html

**Pull request merged**